SPEC.md

```diff
@@ -64,50 +64,55 @@ It includes a *rootfs* with all of the files that will exist in the root of the
 
 ### Image Archives
 
-The ACI archive format aims for flexibility and relies on very boring technologies: HTTP, gpg, tar and gzip.
-This set of formats makes it easy to build, host and secure a container using technologies that are battle-tested.
+The ACI file format ("image archive") aims for flexibility and relies on standard and common technologies: HTTP, gpg, tar and gzip.
+This set of formats makes it easy to build, host and secure a container using technologies that are widely available and battle-tested.
 
-Image archives MUST be a tar formatted file.
-The image may be optionally compressed with gzip, bzip2, or xz.
-After compression, images may also be encrypted with AES symmetric encryption.
+- Image archives MUST be a tar formatted file.
+- All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
+- Image archives MUST be named with the suffix `.aci`, irrespective of compression/encryption (see below).
+- Image archives SHOULD be signed using PGP, in detached signature mode.
```

Comment:
late signing of the _compressed_ image? ...

Uncompressed:
```
vbatts@valse ~ (master *) $ sha256sum ./true.tar
5b921ba77402ea4c618d4e9bafb2031296fde1e3b997ddf274142c51323328bd  ./true.tar
vbatts@valse ~ (master *) $ sha256sum ./true.tar
5b921ba77402ea4c618d4e9bafb2031296fde1e3b997ddf274142c51323328bd  ./true.tar
```

After Compression:
```
vbatts@valse ~ (master *) $ cat true.tar | gzip -9 | sha256sum 
2c5e64edaa32da17d3f0ce8d3a986367fb40c2a6c0f9e230d71f07f62c63ed17  -
vbatts@valse ~ (master *) $ cat true.tar | gzip -9 | sha256sum 
aa244153f70eb6dfb10d012068b7d2499bd7b21e8ba523f242cfb98ddca00522  -
vbatts@valse ~ (master *) $ cat true.tar | gzip -9 | sha256sum 
accd6abd99ff3960aadb13cf83e0a36ee608d3060f5996ba0c8edae8405bcb4f  -
```

if we sign the compressed archive, it'll will largely be a one time sig validation.